jumanji/environments/routing/robot_warehouse/env.py

```diff
@@ -454,7 +454,7 @@ def _update_reward_and_request_queue(
         Returns:
             a random key, updated reward, request queue and shelves.
         """
-        x, y = goal
+        y, x = goal
```

Comment:
Was this an error as well? Why are coordinates inverted? In Jumanji, we usually follow the (x,y) convention instead of the video-game (y,x) one.

Reply:
Yes, this was a bug. The reversal is a quirk from the original environment in terms of how they set up the initial grid. See here on how they all invert when enumerating goal positions: https://github.com/semitable/robotic-warehouse/blob/7b83fe04ec5b57523bc4d6b6c1c179c48849a3f7/rware/warehouse.py#L807